yarn.lock

```diff
@@ -11377,6 +11378,13 @@ send@latest:
   languageName: node
   linkType: hard
 
+"vue-logger-plugin@npm:2.2.3":
```

Comment:
Definitely don't really know how/what exactly needs to be checked in to get a new package working.

Reply:
What do you mean?

Reply:
Just meant that I'm not sure I added the package in all the places it needs to be.

Reply:
The right way to add packages is:
- go within `TERArium/packages/client/hmi-client`
- install package with `yarn add --exact [package_name]`
- make sure `yarn.lock` is part of the commit

Then you are all good, and it looks like this is what you did.

Reply:
gotcha.  Thanks @YohannParis .